examples/unix/c11/z_sub_channel.c

```diff
@@ -64,7 +64,7 @@ int main(int argc, char **argv) {
     }
 
     printf("Declaring Subscriber on '%s'...\n", keyexpr);
-    z_owned_sample_channel_t channel = z_sample_channel_fifo_new(3);
+    z_owned_sample_fifo_channel_t channel = z_sample_fifo_channel(3);
```

Comment:
In zenoh-c we have `z_*_new()`: https://github.com/eclipse-zenoh/zenoh-c/blob/c1498fcaf08d3dad15e49b1ac15cd380c0dc175c/examples/z_queryable_with_channels.c#L57

We can decide to keep the new or to remove it. In any case, zenoh-pico and zenoh-c should be aligned.

Reply:
It seemed to me that I saw a creation without `_new`, but I now double-checked, this is not so, indeed in most cases we have a creation with `_new`, thanks!